testcafe/tests/helpers/data.js

```diff
@@ -7,17 +8,37 @@ export const FOLDER_DATE_TIME = `Folder_${getCurrentDateTime()}`
 export const ALBUM_DATE_TIME = `Album_${getCurrentDateTime()}`
 // data files & path
 const homedir = require('os').homedir
-export const DOWNLOAD_PATH = `${homedir()}/Downloads/`
-export const DOWNLOAD_ZIP_PATH = `${DOWNLOAD_PATH}files.zip`
+export const DOWNLOAD_PATH = `${homedir()}/Downloads`
+export const DOWNLOAD_FOLDER_PATH = `${DOWNLOAD_PATH}/files.zip`
+export const filesList = []
```

Comment:
Would it be possible to have most of this file, but *especially* this variable, in the tests context?

Reply:
No, context (ctx) is use to share variable between test in the same fixture.
When passing some variable from one fixture to another, I need to use the data file 
https://testcafe-discuss.devexpress.com/t/how-to-pass-data-between-tests/693/3

Reply:
Thank you for clarifying why we can't use context for this!

What bothers me is that `fileList` is defined here, modified later, and then used in several tests. I'm really not fond of the coupling this creates, and it requires tests to run in a very specific order to work. I'm less worried about the other constants, since those are defined once and for all.

My understanding is that we want to initialize the list of files before the tests run — if that's the case maybe we could do it in a separate file, so it's not lumped together with other `data`? It's not a perfect solution but it would at least isolate the risky part.

Reply:
The only 'test' that modify `fileList` is `init-data.js` (cf. https://github.com/cozy/cozy-drive/pull/1482/files#diff-4b8d67cbae09254750cc13e491908885R12)  which has to run 1st as it unzip our files.
Other tests using `fileList` are only reading it.

Reply:
Mmh ok. I still feel like there should be a better way to do this, but I don't see it right now.